vhdbuilder/packer/install-dependencies.sh

```diff
@@ -448,11 +459,22 @@ PATCHED_HYPERKUBE_IMAGES="
 1.17.5.1
 1.18.1.1
 1.18.2.1
+1.19.0-beta.1
 "
 for KUBERNETES_VERSION in ${PATCHED_HYPERKUBE_IMAGES}; do
-  CONTAINER_IMAGE="mcr.microsoft.com/oss/kubernetes/hyperkube:v${KUBERNETES_VERSION}"
-  pullContainerImage "docker" ${CONTAINER_IMAGE}
-  echo "  - ${CONTAINER_IMAGE}" >> ${VHD_LOGS_FILEPATH}
+    ## use hyperkube until 1.19
```

Comment:
can you add a TODO to not do this for 1.17/1.18 once we switch them to kube-proxy image in ccp